CefSharp/ModelBinding/DefaultBinder.cs

```diff
@@ -72,6 +72,10 @@ public virtual object Bind(object obj, Type modelType)
                 {
                     throw new ArgumentException("When modelType is an enumerable it must specify the type.", "modelType");
                 }
+            } 
+            else if (modelType.IsGenericObject())
```

Comment:
Type checking here seems a little weak, I think we'd be better off immediately after the `obj == null` check to using something like 

```c#
//If the object can be directly assigned to the modelType then return immediately. 
if(modelType.IsAssignableFrom(obj.GetType()))
{
	return obj;
}
```